README.md

```diff
@@ -38,3 +38,6 @@ proving parts of the appendix or main body of the paper:
 - PartialTraceDefinitions - Adding the functions for tracing out qubits, and
   properties of the function.
 - TraceoutHelpers - Contains helper lemmas for tracing out qubits.
+- DiagonalHelpers.v - Helper lemmas for diagonal matrices.
+- WFHelpers.v - Helper lemmas for well-formed matrices.
```

Comment:
Do you mind going through the files and seeing if there's anything you can say to be more precise, e.g. `WFHelpers: Contains the solve_WF_matrix automation tactic for solving well-formedness goals`

Reply:
Updated the descriptions. Let me know if changes are needed.